entity-types/ext-triton/golden_metrics.yml

```diff
@@ -0,0 +1,48 @@
+nv_cpu_memory_total_bytes:
```

Comment:
We usually use camelCase for Golden metric names and we don't put the dot at the end of the title.